### ISSUE TITLE
Keyboard navigation window

### ISSUE THREAD

**Issue by jolevesq** [Canadian-Geospatial-Platform/geoview#819]
The keyboard navigation window z-index seems wrong. Its opening under the footer bar.

![image.png](https://images.zenhubusercontent.com/5695178c638e718c99600115/8a3494ca-04e3-402d-a9b6-1c63babb243e)

At the same time,

- Resize the window so each sentences it is on its own line

- Add a X button upper right corner and set focus on it on open (think it is WCAG requirement)
- Make sure the theme is used to have black or white window

**Comment by kaminderpal** [comment]
In progress

**Comment by kaminderpal** [comment]
In PR review.
